design/capm3-remediation-controller-proposal.md

```diff
@@ -143,15 +148,16 @@ metadata:
   namespace: NAMESPACE_OF_UNHEALTHY_MACHINE
   ownerReferences:
    - apiVersion: cluster.x-k8s.io/v1alphaX
-     kind: MachineHealthCheck
-     name: REMEDIATION_GROUP
+     kind: Machine
+     name: MACHINE_NAME
 spec:
   type: reboot
-  rebootLimit: 3
+  retryLimit: 3
   timeout: 300s
 status:
   phase: rebooting
```

Comment:
We said above that the `reboot` phase doesn't exist anymore right?

Reply:
This is more for the future work to support multiple remediation strategies. I think `rebootLimit` naming bound us to reboot strategy only.